### PR TITLE
Add ScalaxHack event

### PR DESCRIPTION
**Pull request by davegurnell**
Added an event page for an unconference / hack day we're co-locating with Scala Exchange.

### CODE REVIEW

File: _events/2016-12-10-scalaxhack.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The event is open to the whole community,
+not just attendees of Scala Exchange,
+and will run on 10th December 2016
+(the day after Scala Exchange)
+at [CodeNode][codenode],
+Skills Matter's dedicated conference venue and tech hub
+near Moorgate Station in central London.
+
+The full of sessions and topics
```
Comment (tpolecat) [inline _events/2016-12-10-scalaxhack.md]: full ... list of?

File: _events/2016-12-10-scalaxhack.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+the Typelevel family.
+
+ScalaxHack is a free event.
+Lunch will be sponsored by [Underscore][underscore].
+
+<a class="btn large" href="https://skillsmatter.com/conferences/7975-scalaxhack">Register</a>
+
+<br>
+
+*[Photograph of London skyline][london-skyline] Copyright 2014 John Nuttall, CC-BY-2.0*
```
Comment (larsrh) [inline _events/2016-12-10-scalaxhack.md]: Please add the credits to the file `about.html` below the others.

Comment (davegurnell) [inline _events/2016-12-10-scalaxhack.md]: Will do

**Comment by davegurnell** [comment]
@larsrh I've moved the image attribution to `about.html`. Does that look ok?

**Pull request merged**